docs/website/docs/command-reference/json-output.md

```diff
@@ -157,7 +157,9 @@ odo describe component -o json
 			"containerPort": 3000
 		}
 	],
-	"runningIn": ["Dev"],
+	"runningIn": {
+		"dev": true
```

Comment:
Feature-wise, I was expecting the list of possible running modes to be exposed, instead of just the current running mode.

From what I understand from [this comment](https://github.com/redhat-developer/odo/issues/5999#issuecomment-1206098278):
> menu to start dev mode is enabled if devfile has dev support and if dev mode is not running for the component.
>
If devfile has dev support but dev mode is not running for the component, I think `odo describe component -o json` should have `"dev": false` reported in the running mode object, to help to parse that information. No?

Reply:
Yes, you are right. Writing the doc, I was thinking it was due to the `omitempty` nature of the marshalling in Go, but it is not the case, because it is a map. I'll make the changes to add by default all the keys with a False value